pycbc/waveform/compress.py

```diff
@@ -685,7 +720,8 @@ def write_to_hdf(self, fp, template_hash, root=None):
         else:
             root = '%s/'%(root)
         group = '%scompressed_waveforms/%s' %(root, str(template_hash))
-        for param in ['amplitude', 'phase', 'sample_points']:
+        for param in ['amplitude', 'phase', 
+                      'amplitude_freq_points', 'phase_freq_points']:
```

Comment:
Although this drops by 2 if we can store the interpolant points as floats, not doubles. Which are you using?